cmd/podman/volumes/list.go

```diff
@@ -57,6 +58,9 @@ func list(cmd *cobra.Command, args []string) error {
 	if cliOpts.Quiet && cmd.Flag("format").Changed {
 		return errors.New("quiet and format flags cannot be used together")
 	}
+	if len(cliOpts.Filter) > 0 {
+		lsOpts.Filter = make(map[string][]string)
```

Comment:
Should this be optimized with the len(cliOpts.Filter)?

 		lsOpts.Filter = make(map[string][]string. len(cliOpts.Filter))

Reply:
i think theoretically at some point you can have multiple filters with the same key in which this len assumption would be incorrect.